doc/classes/String.xml

```diff
@@ -459,7 +459,7 @@
 			<argument index="1" name="what" type="String">
 			</argument>
 			<description>
-				Inserts a substring at a given position.
+				Inserts a substring at a given position. Returns a copy of the string.
```

Comment:
This suggests that it modifies the string *and* also returns a copy, while IINM the original string is not modified.